apps/meteor/client/views/home/HomePage.tsx

```diff
@@ -1,11 +1,17 @@
-import { useSetting } from '@rocket.chat/ui-contexts';
+import { useSetting, useCurrentRoute, useRoute } from '@rocket.chat/ui-contexts';
 import type { ReactElement } from 'react';
 import React from 'react';
 
 import CustomHomePage from './CustomHomePage';
 import DefaultHomePage from './DefaultHomePage';
 
 const HomePage = (): ReactElement => {
+	const conferenceRoute = useRoute('conference');
+	const [, params, queryParams] = useCurrentRoute();
+	if (params?.context === 'redirectConference') {
+		conferenceRoute.push(params, queryParams);
+	}
```

Comment:
you should run effects inside effects :)

Reply:
oooooooops